Makefile

```diff
@@ -26,10 +26,10 @@ PROTOC_VERSION = 3.15.6
 PROTOC_GEN_GO_VERSION = 1.26.0
 PROTOC_GEN_GO_GRPC_VERSION = 1.1.0
 
-generate:
-	@go generate $(REPO_PATH)/storage/ent/
+KIND_NODE_IMAGE = "kindest/node:v1.19.7@sha256:a70639454e97a4b733f9d9b67e12c01f6b0297449d5b9cbbef87473458e26dca"
+KIND_TMP_DIR = "/tmp/dex-kind-kubeconfig"
```

Comment:
Can we make this a project local tmp dir instead?

Reply:
Previously, we used `ioutil.TempFile("", "dex-kubeconfig-*")` to generate a tmp file for the config, which was calling `os.TempDir()` undercover. Although, it makes sense to me to move it closer. I will do it.